plugins/chat/lib/steps_inspector.rb

```diff
@@ -30,7 +30,7 @@ def type
       end
 
       def emoji
-        return "❌" if failure?
+        return spec_fail_unexpected_result? ? "❌ 🚧" : "❌" if failure?
```

Comment:
This actually won’t work as the example shown in the screenshots (it should be applied to ✅ too) :thinking: